assets/css/style.css

```
@@ -67,41 +119,169 @@ ul {
   background-position: 0 40px;
   background-size: contain;
   border-bottom-left-radius: 100px;
-  margin-bottom: 2rem;
 }
 
-.hero-content {
+.hero-content-space {
   padding-top: 3rem;
   padding-bottom: 9rem;
 }
 
-.hero h1 {
-  font-weight: 900;
-  font-size: 47px;
-  line-height: 1.4;
+.about-me-content h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
   color: #172b4d;
   margin-top: 90px;
   margin-bottom: 20px;
 }
 
-.hero p {
+.about-me-content p {
   font-size: 18px;
-  font-weight: 500;
+  font-weight: 400;
   color: #344563;
   line-height: 1.7;
 }
 
-.hero .cta {
+.about-me-content .cta {
   color: #7f8cff;
+  margin: 10px 0;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.about-me-content .social {
+  display: flex;
+  justify-content: space-between;
+  width: 200px;
+}
+
+/* work section */
+
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 88px;
+  margin: 114px 0;
+}
+
+.works .item {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  padding: 16px;
+  border-radius: 16px;
+}
```

- [ ] **Good job on making a good design :1st_place_medal: :1st_place_medal:** 
   
   However, It would be better to layout the elements of individual card using ```CSS Flexbox``` :joy: 
It is a good practice to use **CSS Grid** in the main container that contains all of the cards and use **Flexbox** to lay out an individual card. :+1:

---

Thanks for the review @TuyishimireEric . Do you mean i should use add `display:flex` to the individual card like so?
```suggestion
.works .item {
  background-color: #fff;
  border: 1px solid #dfe1e6;
  padding: 16px;
  border-radius: 16px;
  display: flex;
}
```

If that is correct, can you tell me how it affects the layout? so I can learn from it

---

# Hi @mrprotocoll :wave: :wave: 

**Sorry for late responding, :pray:** 
Good job on arranging your contents by using ```display: flex``` 
by default it aligns all the items in left to right, you have to add ```flex-direction: column``` to align them from top to bottom and I happy to see this in your codes :+1: :+1: 

you may follow this [link](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for [A Complete Guide to Flexbox  ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)and I hope it will be more helpful 

## Cheers :clinking_glasses: and happy coding :joy: :joy: